packages/contracts-bedrock/deploy-config/polymer-mainnet.json

```diff
@@ -0,0 +1,20 @@
+{
+  "l1StartingBlockTag": "0x438335a20d98863a4c0c97999eb2481921ccd28553eac6f913af7c12aec04108",
+  "l1ChainID": 1,
+  "l2BlockTime": 2,
+  "l2ChainID": 12345,
+  "l2OutputOracleSubmissionInterval": 1800,
+  "l2OutputOracleStartingTimestamp": 1686068903,
+  "l2OutputOracleStartingBlockNumber": 105235063,
+  "l2OutputOracleProposer": "0x473300df21D047806A082244b417f96b32f13A33",
+  "l2OutputOracleChallenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "finalizationPeriodSeconds": 604800,
+  "proxyAdminOwner": "0x7871d1187A97cbbE40710aC119AA3d412944e4Fe",
+  "useFaultProofs": false,
+  "gasPriceOracleBaseFeeScalar": 1368,
+  "l2GenesisBlockGasLimit": "0x1c9c380",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000010",
```

Comment:
this is optimism's [batcher inbox](https://etherscan.io/address/0xff00000000000000000000000000000000000010). downside of using it is op-node will show lots of unauthorized batch submitter. it's not a security concern but would hinder debug. we can choose a diff address like staging/prod testnets, even though we cannot stop others from sending txs to that address.

Reply:
ah yes, that's a good point. Updated it to be the batcher inbox used in staging for now.